userland/examples/lua-hello/Makefile

```diff
@@ -0,0 +1,20 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../..
+
+# Which files to compile.
+C_SRCS := $(wildcard *.c)
+
+# External libraries used
+EXTERN_LIBS += $(TOCK_USERLAND_BASE_DIR)/lua53
```

Comment:
Then again, it seems like lua updates once every 4-5 years, so maybe this really just doesn't matter